doc/user_manual/existing_interfaces.tex

```diff
@@ -2591,3 +2591,138 @@ \subsubsection{DataObjects}
 \end{lstlisting}
 As mentioned before, this will create $4$ DataObjects.
 %
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%%%%% SERPENT  INTERFACE  %%%%%%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+\subsection{SERPENT Interface}
+\label{subsec:serpentInterface}
+The Serpent interface is meant to run multiple user-defined SERPENT simulations
+while storing the following values: 
+\begin{itemize}
+\item Input composition
+\item Depletion time
+\item Output composition
+\item Beginning Of Cycle / End Of Cycle Keff
+\end{itemize}
+
+It allows users to run SERPENT by varying a value
+(e.g. depletion time, input composition) from a template input file.
+For example, a user can run a \texttt{MultiRun} where the depletion
+time of a fixed input composition is varied, to see the effect of 
+depletion time on the output composition and End Of Cycle Keff.
+
+There are some limitations for this interface:
+\begin{itemize}
+\item Only the parameters mentioned above will be stored in the output csv file
+\item You cannot run multiple depletion steps
+\end{itemize}
+
+Note that due to the number of isotopes to track,
+we use the \xmlnode{ExternalXML} to import from a file.
+The user must edit the \texttt{aux-input-files} so that it
```

Comment:
the aux-input-file, should not be committed and added in ```framework/CodeInterfaces/SERPENT```. It is an input and should be explained here and placed in the tests folder